docs/howto/jsonapi/custom_type_value.md

```diff
@@ -0,0 +1,25 @@
+# How to customize the type of a JSON API document
+
+To change the type of a resource, just use the `type` class method in your serializer definition.
+
+Note that by default, type is pluralized.
```

Comment:
when I just used type, if you specify 'post' it comes out as 'post' in the json -- so whatever is passed  to the type method is what gets rendered. So, I do think that it's worth mentioning. maybe it could be worded better, though.